client.go

```diff
@@ -341,6 +369,24 @@ func (client *Client) HOCRText() (out string, err error) {
 	return
 }
 
+func (client *Client) DetectOrientationScript() (int, float32, string, float32, error) {
+	if err := client.initOsd(); err != nil {
+		return 0, 0, "", 0, err
+	}
+	var (
+		orient_deg  C.int
+		orient_conf C.float
+		script_name *C.char
```

Comment:
🤔 thinking about memory leak

Reply:
fixed in: https://github.com/otiai10/gosseract/pull/299/commits/d0f75aa03d096251a845413699d8a6fe82407cc4